pg_bm25/src/index_access/build.rs

```diff
@@ -127,35 +98,22 @@ unsafe extern "C" fn build_callback(
 unsafe extern "C" fn build_callback_internal(
```

Comment:
Since we're cleaning up the build_callback...this doesn't need to be extern C does it?